ui/v2.5/src/core/StashService.ts

```diff
@@ -830,6 +830,8 @@ export const useTagsDestroy = (input: GQL.TagsDestroyMutationVariables) =>
     update: deleteCache(tagMutationImpactedQueries),
   });
 
+export const useSceneSaveActivity = () => GQL.useSceneSaveActivityMutation();
```

Comment:
```suggestion
export const useSceneSaveActivity = () => 
  GQL.useSceneSaveActivityMutation({
    update: deleteCache(sceneMutationImpactedQueries),
  });
```

To fix the stale cache issue.

Reply:
Ahh, thanks for this. I'll need to remember this going forwards.